kstreams/exceptions.py

```diff
@@ -19,6 +19,20 @@ def __str__(self) -> str:
         return msg
 
 
+class StreamNotFoundException(StreamException):
```

Comment:
I honestly don't need the StreamNotFoundException, because python list.remove will throw an error as well:
```If the element doesn't exist, it throws ValueError: list.remove(x): x not in list exception.``` Thoughts on this? @marcosschroh @woile